librz/asm/arch/hexagon/hexagon_arch.h

```diff
@@ -42,7 +42,7 @@ typedef struct {
 
 #define HEX_PKT_UNK           "?   "
 #define HEX_PKT_SINGLE        "[   "
-#define HEX_PKT_SINGLE_UTF8   "[     "
+#define HEX_PKT_SINGLE_UTF8   "[   "
```

Comment:
Could you extract this change and Doxygen comments to the Hexagon structures into a separate PR? We could merge it before the 0.4.0 release even.

Reply:
Unfortunately no. The current coloring function removes some spaces after the `[` or adds some after the UTF-8 brackets (or something similar, can't remember at the moment). If we merge this change now the assembly would be misaligned in visual mode:

```
     ┌     immext(##sym.__cxa_finalize) 
     └     R0 = ##sym.__cxa_finalize 
     [   P0 = cmp.eq(R0,##0x0)
     [   if (P0) jumpr:nt LR   
```

The new coloring method leaves the asm string as is, which makes the deletion of spaces necessary for alignment.